### PR TITLE
return current user from main method

### PR DESCRIPTION
**Pull request by SmbatYeranyan**

### CODE REVIEW

File: src/main.js [hunk 1]
```diff
@@ -73,6 +73,10 @@ default {
         return config;
     },
 
+    getCurrentUser: function() {
```
Comment (jontore) [inline src/main.js]: No test :)

Comment (mcgnly) [inline src/main.js]: tests added

**Comment by jontore** [comment]
👍

**Comment by mcgnly** [comment]
does thumbs up mean I should merge it?

**Comment by jontore** [comment]
Yes :)

**Pull request merged**